packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/web_gesture_handling.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// This setting controls how the API handles gestures on the map
+enum WebGestureHandling {
```

Comment:
Have you considered making this an "enhanced enum" backed by a `String`? That should cut down in the use of `.name` below, right?

Reply:
Enhanced enums let you override a `toString()` method that ends up used as expected in the JS code, see [this](https://github.com/flutter/packages/blob/main/packages/google_identity_services_web/lib/src/js_interop/shared.dart#L23-L40) as an example.

Here's some [Enhanced Enums Docs](https://dart.dev/language/enums#declaring-enhanced-enums).

But it's all right, let's leave this one as is.

Reply:
Right it is possible that way you just worded it weird 🙃 
If you're fine leaving it that's fine with me